client/reader/stream/index.jsx

```diff
@@ -481,25 +424,50 @@ class ReaderStream extends React.Component {
 					) }
 
 				<UpdateNotice
-					count={ this.state.updateCount }
+					count={ updateCount }
```

Comment:
can we connect up updateNotice and pass it the stream key?

Reply:
i'll look into it! seems like that should work to me